### PR TITLE
fix `Wrong svg element processing in IE` (close #1083)

### LINKED ISSUE #1083: Wrong svg element processing in IE
Reproduced in playground in IE11:

1) Open [DevExtreme Charts demo page](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/AjaxRequest/jQuery/Light/) in playground
2) Export chart image:
    a) click here: 

    b) select "PNG file"
3) The `SyntaxError` error is appeared when the following code is called:
```js
var s = new DOMParser();
s.parseFromString(e, "image/svg+xml")
```
Without hammerhead processing `e` looks like:
```html
<svg xmlns="someurl" version="1.1">...</svg>
```
But with processing it looks like:
```html
<svg xmlns="someurl" version="1.1" xmlns:NS1="" NS1:xmlns:ns1="">...</svg>
```
It brokes the `parseFromString` function

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
ping @churkin @miherlosev

File: src/client/utils/html.js [hunk 1]
```diff
@@ -213,3 +220,25 @@ export function processHtml (html, parentTag, prepareDom) {
         return true;
     });
 }
+
+function removeExtraNamespaces (html, processedHtml) {
```
Comment (miherlosev) [inline src/client/utils/html.js]: `removeExtraSvgNamespeces`

File: test/client/fixtures/sandbox/code-instrumentation/getters-test.js [hunk 2]
```diff
@@ -481,3 +481,37 @@ test('we should clean up hammerhead script and style for all elements(GH-1079)',
     ok(checkProperty(innerHTML));
     ok(checkProperty(outerHTML));
 });
+
+if (browserUtils.isIE && !browserUtils.isMSEdge) {
+    test('we should clean up html and remove extra namespaces from svg (GH-1083)', function () {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/code-instrumentation/getters-test.js]: Change checking algorithm:
1)create native element, set `xmlns:link` attribute for this, calculate href for `a` tag
2)create overriden element, set ....
3)check that native element markup equals the processed markup

**Comment by LavrovArtem** [comment]
FPR

File: test/client/fixtures/sandbox/code-instrumentation/getters-test.js [hunk 3]
```diff
@@ -481,3 +481,39 @@ test('we should clean up hammerhead script and style for all elements(GH-1079)',
     ok(checkProperty(innerHTML));
     ok(checkProperty(outerHTML));
 });
+
+if (browserUtils.isIE) {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/code-instrumentation/getters-test.js]: As I understand we can remove this condition

Comment (LavrovArtem) [inline test/client/fixtures/sandbox/code-instrumentation/getters-test.js]: yep

**Comment by LavrovArtem** [comment]
FPR

**Comment by LavrovArtem** [comment]
ping

File: src/client/utils/html.js [hunk 4]
```diff
@@ -213,3 +220,25 @@ export function processHtml (html, parentTag, prepareDom) {
         return true;
     });
 }
+
+function removeExtraSvgNamespeces (html, processedHtml) {
+    var initialSvgStrs = html.match(FIND_SVG_RE);
```
Comment (miherlosev) [inline src/client/utils/html.js]: Do fast return if html doesn't contains svg tags (as in https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/node/index.js#L132)

File: test/client/fixtures/sandbox/code-instrumentation/getters-test.js [hunk 5]
```diff
@@ -481,3 +481,37 @@ test('we should clean up hammerhead script and style for all elements(GH-1079)',
     ok(checkProperty(innerHTML));
     ok(checkProperty(outerHTML));
 });
+
+test('we should clean up html and remove extra namespaces from svg (GH-1083)', function () {
+    var svg       = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/code-instrumentation/getters-test.js]: You use reqular expession so need to add test cases for all variants:
`innerHtml` doesn't contains `svg` tag
`innerHtml` contains two `svg` tags
`svg` tag without `NS` attribute
`svg` tag with two `NS` attribute

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
FPR

**Comment by LavrovArtem** [comment]
ping @churkin

**Pull request merged**